CONTRIBUTING.md

```diff
@@ -19,28 +19,20 @@ branch of your fork and then submit a pull request.
 
 ## Running the tests
 
-We use [nose](https://nose.readthedocs.org/en/latest/) for testing. To run the
+We use [pytest](https://docs.pytest.org/en/latest/) for testing. To run the
 tests first install nose using pip:
 
-`pip install nose`
+`pip install pytest`
 
 Then from the root of the repository install the Retriever:
 
 `python setup.py install`
 
 and run the tests:
 
-`nosetests`
+`pytest test`
```

Comment:
@stellargo Thank you for contributing to the retriever. Change `pytest test`  to `pytest`